draft-ietf-quic-multipath.md

```diff
@@ -889,31 +889,33 @@ Terrestrial | 100ms  | 350ms
 Satellite   | 350ms  | 600ms
 {: #fig-example-ack-delay title="Example of ACK delays using multiple paths"}
 
-Using the default algorithm specified in {{QUIC-RECOVERY}} would result
-in suboptimal performance, computing average RTT and standard
-deviation from series of different delay measurements of different
-combined paths.  At the same time, early tests showed that it is
-desirable to send ACKs through the shortest path because a shorter
-ACK delay results in a tighter control loop and better performances.
-The tests also showed that it is desirable to send copies of the ACKs
-on multiple paths, for robustness if a path experiences sudden losses.
-
-An early implementation mitigated the delay variation issue by using
-time stamps, as specified in {{QUIC-Timestamp}}.  When the timestamps
-are present, the implementation can estimate the transmission delay
-on each one-way path, and can then use these one way delays for more
-efficient implementations of recovery and congestion control
-algorithms.
-
-If timestamps are not available, implementations could estimate one
-way delays using statistical techniques.  For example, in the example
-shown in Table 1, implementations can use "same path"
-measurements to estimate the one way delay of the terrestrial path to
-about 50ms in each direction, and that of the satellite path to about
-300ms.  Further measurements can then be used to maintain estimates
-of one way delay variations, using logical similar to Kalman filters.
-But statistical processing is error-prone, and using time stamps
-provides more robust measurements.
+The ACK_MP frames describe packets that were sent on the specified path,
+but they may be received through any available path. There is an
+understandable concern that if successive acknowledgements are received
+on different paths, the measured RTT samples will fluctuate widely,
+and that might result in poor performance. In fact, this concern is
+probably not justified.
+
+The computed values reflect both the state of the network path and the
+scheduling decisions by the sender of the ACK_MP frames. In the example
+above, we may assume that the ACK_MP will be sent over the terrestrial
+link, because that provides the best response time. In that case, the
+computed RTT value for the satellite path will be about 350ms. This
+lower than the 600ms that would be measured if the ACK_MP came over
+the satellite channel, but it is still the right value for computing
+for example the PTO timeout: if an ACK_MP is not received after more
+than 350ms, either the data packet or its ACK_MP were probably lost.
```

Comment:
Isn't the PTO 3xRTT?

Reply:
However, I'm not sure about this. In this examples you assume that the terrestrial path will used in a stable way for ACK. In the paragraph above you say if different path are used which is a different case. So if I send half my ACK over each pass, I will see an average RTT of 475ms which is too low for those ACKs that are sent over the satellite path.

Reply:
Commenting on this PR after the merge: in the example that you cite, the smoothed RTT will actually be very close to the shortest path RTT + maybe 1 ACK delay. ACK_MP frames are redondant. If you send ACK alternately on short and long path, the ACK on the short path will arrive before the one on the long path. When the ACK on the long path arrives, its "highest" packet will already have been acked as part of a range carried on the short path, so it will not contribute to RTT computation.

Reply:
Only if you send ACKs multiple time per RTT and the latency difference between the path is large.

So maybe we should rather not make any default recommendations but just explain the problems and non-problems in different alternatives...?

(btw. which merge are you talking about? This PR is not merged yet)

Reply:
From RFC 9002:
```
PTO = smoothed_rtt + max(4*rttvar, kGranularity) + max_ack_delay
```